modules/images-editing-image-registry-settings-hcp.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="images-editing-image-registry-settings-hcp_{context}"]
+= Editing image registry settings for {hcp-title}
+
+You can change the image registry config with the `rosa edit` command.
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, are blocked unless explicitly listed. To prevent pod failure, a list of Red Hat registries is automatically whitelisted, as they are required by payload images within your environment. The current list consists of `image-registry.openshift-image-registry.svc:5000,quay.io,registry.redhat.io` and it is also visible when running `rosa describe cluster`.
+====
+
+[NOTE]
+====
+You can change any registry related parameter which will trigger a rollout across all machinepools (all machinepool nodes will be recreated, following pod draining from each node).
+====
+
+.Procedure
+
+* Update or edit the image registry for the cluster by running the following command:
+
++
+[source,terminal]
+----
+$ rosa edit cluster --registry-config-insecure-registries <insecure_registries> \
+   --registry-config-blocked-registries <insecure_registries> \
+   --registry-config-allowed-registries <allowed_registries> \
```

Comment:
The two flags are mutually exclusive.We can just edit with one of them.

 --registry-config-blocked-registries <insecure_registries> \
   --registry-config-allowed-registries <allowed_registries> \

Reply:
removed "  --registry-config-blocked-registries <insecure_registries> \"